### PR TITLE
Update user guide in line with PE-D bug reports

### LINKED ISSUE #176: [PE-D][Tester B] User Guide and Program Spelling Error
There is a spelling error under the User Guide's menu section. It should've been spelled as "event", not "even" (Last line of image).

That being said, this spelling error translates to the menu command in the actual jar file program.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: Holy-An/ped#1

### LINKED ISSUE #177: [PE-D][Tester B] Spelling Error in User Guide Bug
There is a spelling error under the User Guide's quick start section. It should have been spelled as "Download", not "Down" (Last line of image).

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: Holy-An/ped#2

### LINKED ISSUE #179: [PE-D][Tester A] Inconsistent naming of JAR file in documentation
In the User Guide, the jar file is referenced to as "EventManagerCLI.jar". However, the actual JAR file is named "manager.jar".

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: Dri-water/ped#2

### LINKED ISSUE #199: [PE-D][Tester C] Minor error in UG
This command can also view items instead of just participants. Might want to update to make it clearer

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: KHsienKit/ped#4

### PR DESCRIPTION
**Pull request by jemehgoh**
This PR comprises updates to the DG in line with bug reports, for consistency, and to reflect the program's functionality.

It excludes amendments to the `edit` command and to the storage functionality.

Fixes #176.
Fixes #177.
Fixes #179.
Fixes #199.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -5,18 +5,18 @@
...
-  * [Mark an event as done: `mark`](#mark-an-event-as-done-mark)
-  * [Marks a participant as present: `mark`](#marks-a-participant-as-present-mark)
-  * [Marks an item as accounted for: `mark`](#marks-an-item-as-accounted-for-mark)
-  * [Copies participant list: `copy`](#copies-participant-list-copy)
-  * [Sorts event list: `sort`](#sorts-event-list-sort)
-  * [Filters event list: `filter`](#filters-event-list-filter)
-  * [Find participant: `find`](#find-participant-find)
+  * [Listing all events: `list`](#listing-all-events-list)
+  * [Adding an event, participant or item: `add`](#adding-an-event-participant-or-item-add)
+  * [Removing an event or participant: `remove`](#removing-an-event-or-participant-remove-)
```
Comment (KuanHsienn) [inline docs/UserGuide.md:10]: can include remove item

File: docs/UserGuide.md [hunk 2]
```diff
@@ -5,18 +5,18 @@
...
+  * [Viewing all participants or items for an event: `view`](#viewing-all-participants-or-items-for-an-event-view)
+  * [Editing the information of an event or a participant: `edit`](#editing-the-information-of-an-event-or-a-participant-edit)
```
Comment (KuanHsienn) [inline docs/UserGuide.md:12]: same for this

File: docs/UserGuide.md [hunk 3]
```diff
@@ -5,18 +5,18 @@
...
+  * [Marking an event as done: `mark`](#marking-an-event-as-done-mark)
+  * [Marking a participant as present: `mark`](#marking-a-participant-as-present-mark)
+  * [Marking an item as accounted for: `mark`](#marking-an-item-as-accounted-for-mark)
```
Comment (KuanHsienn) [inline docs/UserGuide.md:15]: if for add, remove and edit we combine all three to one line, we should do the same for this,

File: docs/UserGuide.md (hunk 1 above)
Comment (jemehgoh) [reply docs/UserGuide.md:10]: Noted, will change accordingly.

File: docs/UserGuide.md (hunk 3 above)
Comment (jemehgoh) [reply docs/UserGuide.md]: I've combined the three sections into one section. You can review it if it flows well or not.

**Pull request merged**